src/types/test-failed/override-for-non-existing-function-in-trait-without-inheritance.tact

```diff
@@ -0,0 +1,5 @@
+trait BaseTrait {}
+
+contract Test {
```

Comment:
This should be `trait Test`, right? The file is called `override-for-non-existing-function-in-trait-without-inheritance.tact`

Reply:
oops, fixed